nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -577,6 +565,43 @@ def test_add_contours():
     assert len(figure.figure.to_dict().get("data")) == 4
 
 
+@pytest.fixture
+def surface_image_roi():
+    """SurfaceImage for plotting."""
+    mesh, roi_map, _ = _generate_data_test_surf_roi()
+    mesh = InMemoryMesh(coordinates=mesh[0], faces=mesh[1])
+    surf_map = SurfaceImage(
+        mesh={"left": mesh, "right": mesh},
+        data={"left": roi_map, "right": roi_map},
+    )
+    return surf_map
```

Comment:
could not make it work with one of the base surface image fixture, so created another one here

probably worth refactoring after the release